### PR TITLE
Capistrano tasks for db backups (#1051)

### PR DESCRIPTION
**Pull request by sethetter**
* `cap stagename db:backup:create` - creates a backup and stores it on the server
* `cap stagename db:backup:fetch` - creates a backup and downloads it locally

### CODE REVIEW

File: server/app/Console/Commands/DatabaseBackup.php [hunk 1]
```diff
@@ -55,13 +55,15 @@ public function setup($backups_path)
     //Run mysqldump and include timestamp in file name
     public function backup($backups_path)
     {
-        $creds = yaml_parse_file(app_path().'/config/creds.yml');
-        $timestamp = date('c', strtotime('now'));
+        //$creds = yaml_parse_file(app_path().'/config/creds.yml');
```
Comment (doshitan) [inline server/app/Console/Commands/DatabaseBackup.php]: Why comment out instead of delete?

File: server/app/Console/Commands/DatabaseBackup.php [hunk 2]
```diff
@@ -55,13 +55,15 @@ public function setup($backups_path)
...
+        $timestamp = date('YmdHis', strtotime('now'));
         $filename = $timestamp.'_bak.sql';
+        $backup_target = $backups_path.'/'.$filename;
 
-        exec('mysqldump '.$creds['database'].' -u'.$creds['username'].' -p'.$creds['password'].' > '.$backups_path.'/'.$filename);
+        //exec('mysqldump '.$creds['database'].' -u'.$creds['username'].' -p'.$creds['password'].' > '.$backups_path.'/'.$filename);
```
Comment (doshitan) [inline server/app/Console/Commands/DatabaseBackup.php]: Same as above?

**Comment by sethetter** [comment]
@doshitan: removed the commented out code.

**Comment by doshitan** [comment]
:+1:

**Pull request merged**